Project.toml

```diff
@@ -3,13 +3,13 @@ uuid = "1277b4bf-5013-50f5-be3d-901d8477a67a"
 repo = "https://github.com/JuliaArrays/ShiftedArrays.jl.git"
 version = "1.0.0"
 
-[deps]
 
 [compat]
 julia = "1"
 
 [extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
```

Comment:
Could we only add AbstractFFTs on testing? FFTW comes with some binary dependency that I'd prefer to avoid.